go/src/oauth2client/oauth2client.go

```diff
@@ -213,8 +207,7 @@ func (c ThreeLeggedClient) GetToken(scope string) (*Token, error) {
 	return verifyFlow(c.secret, scope, code)
 }
 
-// Run the two-legged oauth flow, will create a JWT token and use the JWT token
-// to fetch an oauth token. Returns the token Object
+// Run 2LO flow. Create a JWT token and use it to fetch an OAuth token.
```

Comment:
The Go tradition for comment for a public function starts with the function name.
e.g. GetToken runs 2LO flow. ...